content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/clientlibs/site/js/carousel.js

```diff
@@ -377,11 +477,14 @@
          * @private
          */
         function resetAutoplayInterval() {
-            if (!that._properties.autoplay) {
+            if (that._paused || !that._properties.autoplay) {
                 return;
             }
             clearAutoplayInterval();
             that._autoplayIntervalId = window.setInterval(function() {
+                if (document.visibilityState && document.hidden) {
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API. Here we prevent the interval from pinging a navigation if the page is hidden (tab is background, minimised or OS lock).